index.html

```diff
@@ -1797,23 +1775,22 @@
                 </li>
                 <li>
                   <a>Queue a task</a> to <a>fire</a> an event named
-                  <code>connectionavailable</code> at <a data-lt=
-                  "PresentationReceiver"><code>presentation.receiver</code></a>.
+                  <code>connectionavailable</code> at all
+                  <a>PresentationConnectionList</a>.
                 </li>
               </ol>
             </li>
           </ol>
           <p>
             A <a>receiving user agent</a> MUST fire an event named
-            <code>connectionavailable</code> on a <a>PresentationReceiver</a>
-            when a connection associated with the object is created. It is
-            fired at the <a>PresentationReceiver</a> instance, using the
-            <a>PresentationConnectionAvailableEvent</a> interface, with the
-            <a for="PresentationConnectionAvailableEvent">connection</a>
-            attribute set to the <a><code>PresentationConnection</code></a>
-            object that was created. The event is fired for all connections
-            that are created for the <a>presentation</a> when <a>monitoring
-            incoming presentation connections</a>.
+            <code>connectionavailable</code> on all
+            <a>PresentationConnectionList</a> when a connection is created. It
```

Comment:
Similar comment about firing the event at a single intended PresentationConnectionList.